telemetry_examples/src/main/java/com/newrelic/telemetry/examples/SpanExample.java

```diff
@@ -49,6 +50,7 @@ public static void main(String[] args) throws Exception {
             .httpPoster(new OkHttpPoster(Duration.ofSeconds(5)))
             .attributesJson(new AttributesGson(new Gson()))
             .enableAuditLogging()
+            .uriOverride(URI.create("https://staging-trace-api.newrelic.com"))
```

Comment:
I'm thinking that we don't want our examples pointing at staging, we should just omit this for clarity and brevity.

Reply:
👍

Reply:
removed